src/site/pages/modifiers.md

```diff
@@ -28,13 +28,11 @@ title: Modifiers
 - pf-m-flex-3
 - pf-m-flex-4
 - pf-m-flex-5
-- pf-m-focus
```

Comment:
wow, forgot this file existed!

Reply:
only found it when i did a search in the repo

Reply:
It creates this page, which I don't know if anyone uses https://pf4.patternfly.org/modifiers